_posts/2021-03-16-galaxy-21-01.md

```diff
@@ -0,0 +1,37 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+tags: [release]
+title: UseGalaxy.eu update to 21.01
+supporters:
+ - denbi
+ - elixir
+---
+
+UseGalaxy.eu has been updated to 21.01, a few days before the official release. Enjoy the preview!
+
+
+## Some Highlights
+
+### New History Panel (Beta)
+  This release includes a preview of the new History Panel. The new history panel can be enabled and disabled in the options menu -> Use Beta History Panel.
+
+### Performance Improvements
+  This release includes many performance improvements for browsing and searching Data libraries (Many thanks to `Oleg Zharkov <https://github.com/OlegZharkov>`__, `Pull Request 10204`_, `Pull Request 11099`_, `Pull Request 10557`_, `Pull Request 10558`_, `Pull Request 10807`_),
```

Comment:
not sure what's going on here, but it's displayed in a strange way, might be the double _?

Reply:
could be. let me fix it

Reply:
I don't have yet the build environment in the new notebook. Can you try again?

Reply:
Isn't that RST syntax and we are here in Markdown?

Reply:
looks good now, at least in the GitHub interface :)